docs/sources/enterprise/access-control/assign-rbac-roles.md

```diff
@@ -79,70 +79,42 @@ Instead of using the Grafana role picker, you can use file-based provisioning to
-
-The following example assigns the `users:writer` fixed role to the `user editors` and `user admins` teams:
-
-```yaml
-# config file version
-apiVersion: 1
-
-# Roles to insert/update in the database
-roles:
-  - name: fixed:users:writer
-    global: true
-    teams:
-      - name: 'user editors'
-        orgId: 1
-      - name: 'user admins'
-        orgId: 1
-```
-
-</br>
-
-**To assign a custom role to a team:**
-
-1. Open the YAML configuration file.
-
-1. Refer to the following table to add attributes and values.
-
-   | Attribute     | Description                                                                                                                                                                                                                                                  |
-   | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-   | `name`        | Enter the name of the custom role.                                                                                                                                                                                                                           |
-   | `version`     | Enter the custom role version number. Assignments are updated if the version of the role is greater then or equal to the version number stored internally. If you are updating a role assignment, you are not required to increment the role version number. |
-   | `global`      | Enter `true` or `false`                                                                                                                                                                                                                                      |
-   | `permissions` | Enter the permissions `action` and `scope` values. For more information about permissions actions and scopes, refer to [LINK]                                                                                                                                |
-   | `teams`       | Enter the team or teams to which you are adding the custom role.                                                                                                                                                                                             |
-   | `orgId`       | Because teams belong to organizations, you must add the `orgId` value.                                                                                                                                                                                       |
+   <!-- TODO find out what LINK was meant to go there -->
```

Comment:
No link is meant to go here. The step is telling the user to refer to (or use) the information in the table to add attributes and values.

Reply:
I was referring to this line, that was previously in the docs with a `[LINK]` placeholder:
```md
Enter the permissions `action` and `scope` values. For more information about permissions actions and scopes, refer to [LINK]
```

Reply:
Figured it out!